zipline/finance/performance/tracker.py

```diff
@@ -319,10 +319,13 @@ def process_event(self, event):
                 midnight = event.dt
 
             if midnight not in self.all_benchmark_returns.index:
-                raise AssertionError(("Date %s not allocated in  "
-                                      "all_benchmark_returns. Calendar "
-                                      "seems to mismatch with benchmark."
-                                      % midnight))
+                raise AssertionError(
+                    ("Date %s not allocated in all_benchmark_returns. "
+                     "Calendar seems to mismatch with benchmark."
+                     "Last dt in container is=%s" %
+                     (midnight,
+                      self.all_benchmark_returns.index)))
```

Comment:
Yes, I neglected to push the fixup before PR'ing. Mea culpa. (Was originally just the last, but figured it was better to get the `__str__` which will show both beginning and end.)